csharp/src/Apache.Arrow/Arrays/ArrowArrayFactory.cs

```diff
@@ -25,6 +25,8 @@ public static IArrowArray BuildArray(ArrayData data)
         {
             switch (data.DataType.TypeId)
             {
+                case ArrowTypeId.Null:
```

Comment:
Can we add some tests for the Null array?

Reply:
I should be able to add tests within the next day.